app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -118,8 +123,13 @@
         reportType = org.acra.sender.HttpSender.Type.JSON,
         httpMethod = org.acra.sender.HttpSender.Method.PUT)
 public class CommCareApplication extends Application {
+
     private static final String TAG = CommCareApplication.class.getSimpleName();
 
+    // Tracking ids for Google Analytics
+    private static final String LIVE_TRACKING_ID = "UA-69708208-1";

```

Comment:
hm, do we want these to get injected by gradle? Seems bad for us to have them be public TBH.